js/src/components/experience-rating-banner/index.js

```diff
@@ -9,9 +9,37 @@ import { external as externalIcon } from '@wordpress/icons';
  * Internal dependencies
  */
 import AppButton from '../app-button';
+import { REPORT_SOURCE_PAID } from '~/constants';
+import useMCProductStatistics from '~/hooks/useMCProductStatistics';
+import useGoogleMCAccount from '~/hooks/useGoogleMCAccount';
+import useProductsReport from '~/pages/reports/products/useProductsReport';
 import './index.scss';
 
 const ExperienceRatingBanner = () => {
+	const { data: statisticsData } = useMCProductStatistics();
+	const { isReady: isMCAccountReady } = useGoogleMCAccount();
+	const { data: productsReport } = useProductsReport( REPORT_SOURCE_PAID );
+
+	const totalProducts = Object.values(
+		statisticsData?.statistics || {}
+	).reduce( ( total, value ) => total + value, 0 );
+
+	const notSyncedProducts = statisticsData?.statistics?.not_synced || 0;
+	const activeProducts = statisticsData?.statistics?.active || 0;
+	const conversions = productsReport?.totals?.conversions?.value || 0;
+
+	const isApproved = ( activeProducts / totalProducts ) * 100 >= 70;
```

Comment:
@ankitrox Let's use a more descriptive name for the constant.
```suggestion
	const hasEnoughActiveProducts = ( activeProducts / totalProducts ) * 100 >= 70;
```